packages/ui/src/components/va-carousel/VaCarousel.vue

```diff
@@ -120,6 +138,8 @@ export default defineComponent({
     } = useCarousel(props, currentSlide)
 
     const { withPause, computedSlidesStyle, slides } = useCarouselAnimation(props, currentSlide)
+    const isObjectSlides = props.items.length && props.items.every((el) => typeof el === 'object' && !!el.src)
```

Comment:
Just a tip - pretty sure that you know that `typeof` `null` will return 'object' as well.
```suggestion
    const isObjectSlides = props.items.length && props.items.every((el) => typeof el === 'object' && !!el?.src)
```

Reply:
shouldn't `isObjectSlide` be `computed`?